src/test/java/com/um/inventoryservice/DataLayer/InventoryItemRepositoryTest.java

```diff
@@ -31,6 +35,32 @@ public void insertInventoryItem (){
                 .verifyComplete();
     }
 
+    @Test
+    public void getAllInventoryItems() {
+        InventoryItem invItem = buildInventoryItem();
+
+        Publisher<InventoryItem> setup = inventoryItemRepository.deleteAll().thenMany(inventoryItemRepository.save(invItem));
+
+        StepVerifier
+                .create(setup)
+                .expectNextCount(1)
+                .verifyComplete();
+
+        Flux<InventoryItem> find = inventoryItemRepository.findAll();
+        Publisher<InventoryItem> composite = Mono
+                .from(setup)
+                .thenMany(find);
+
+        StepVerifier
+                .create(composite)
+                .consumeNextWith(foundInvItem ->{
+                    assertEquals(invItem.getInventoryItemId(), foundInvItem.getInventoryItemId());
+//                    assertEquals(invItem.getStockItemDTO(), foundInvItem.getStockItemDTO());
```

Comment:
do we need this commented line?

Reply:
The reference address for both StockItems will always be different, therefore the test will always fail with that assertEquals statement. The line will be removed.